File: javascript/chapter7/multiples_of_ten.js
const number = prompt("Enter a number: ");

// Trim all spaces of the input and convert to number
if (Number.isNaN(number?.trim())) {
	alert("Invalid number");
	window.close();
}

if (Number(number?.trim()) % 10 !== 0) {
	alert(`${number} is not a multiple of 10!`);
} else {
	alert(`${number} is a multiple of 10!`);
}

File: python/chapter7/multiples_of_ten.py
try:
    number = int(input("Enter a number: "))
except ValueError:
    print("Invalid number")
    exit(1)
else:
    if number % 10 != 0:
        print(f"{number} is not a multiple of 10!")
    else:
        print("{number} is a multiple of 10!")
